cmd/controller/osdisk.go

```diff
@@ -0,0 +1,104 @@
+/*
```

Comment:
@kmova This file contains functions to get uuid using mounts file and mount point. This is not related to controller. Thinking where to put this file inside util or some other package ... your suggestion